helm/hpcc/templates/roxie.yaml

```diff
@@ -328,14 +328,14 @@ spec:
   template:
     metadata:
       labels:
+{{ include "hpcc.addStandardLabels" (dict "root" $ "component" ( printf "%s-agent" $roxie.name ) "name" $roxie.name "instance" $name) | indent 8 }}
```

Comment:
I don't think only "instance" should be dynamic - should always be component=roxie-agent (as used elsewhere), and name should be "roxie" I think.